### DISCUSSION TITLE
Status of Entities on Home

### DISCUSSION THREAD

**Discussion by thesebastianf** [Blackymas/NSPanel_HA_Blueprint#414] [Q&A] [answered]
Likely I am just to blind to see ... however I can't find out how ...
On the "Home" page I would like to see the status of an entity:
e.g. the status of garage door is open or closed ; ideally also with a icon for open and close state

Is this possible to place it in the highlighted area?
![image](https://user-images.githubusercontent.com/13987575/224400964-2f315a19-f187-4f02-a11f-33e38e62aeb2.png)

**Comment by Blackymas** [comment, marked as answer]
It would be possible with the next Version 3.2
